### ISSUE TITLE
About the original experiments data

### ISSUE THREAD

**Issue by xuyan333** [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning#1]
I'm tring to follow your steps in paper2, but I‘m not clear about the raw data for image_iatrix.m,could you please add the data samples in order to help me understand this excellent method better?

**Comment by CQUPanda** [comment]
There must be some mistakes that I'm not the author of mentioned paper but I'm interested in this paper as well.




------------------ 原始邮件 ------------------
发件人: "xuyan333";<notifications@github.com>;
发送时间: 2017年10月30日(星期一) 晚上8:50
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "Subscribed"<subscribed@noreply.github.com>; 
主题: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning]About the original experiments data (#1)




I'm tring to follow your steps in paper2, but I‘m not clear about the raw data for image_iatrix.m,could you please add the data samples in order to help me understand this excellent method better?

**Comment by zhangweiqiang285** [comment]
I also don't clear about the raw data for image_matrix.m. Is there any samples or labels to help us to understand? Thanks a lot.

**Comment by HustWolfzzb** [comment]
大哥，看你的名字，输中文应该没问题。
是这样的，小弟（我还在读大四，应该比你小吧~）做毕设也是选择的故障诊断方面的，但是目前决定采用决策树算法来基于数据驱动。现在苦于找不着数据来源。看你的这个Project，应该也是下了很多功夫的。不知道能否指点一下小弟，哪些地方能找到机械设备运行时候的大量数据的？如果哥你那儿有，能发我一份，那就更加感激不尽了！

**Comment by CQUPanda** [comment]
附件文献用的西储大学轴承故障数据，参考文献有数据地址。




------------------ 原始邮件 ------------------
发件人: "张照博"<notifications@github.com>;
发送时间: 2018年4月3日(星期二) 下午4:46
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "重庆大学机械学院孙攀"<562251022@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning]About the original experiments data (#1)




大哥，看你的名字，输中文应该没问题。
 是这样的，小弟（我还在读大四，应该比你小吧~）做毕设也是选择的故障诊断方面的，但是目前决定采用决策树算法来基于数据驱动。现在苦于找不着数据来源。看你的这个Project，应该也是下了很多功夫的。不知道能否指点一下小弟，哪些地方能找到机械设备运行时候的大量数据的？如果哥你那儿有，能发我一份，那就更加感激不尽了！

**Comment by QiuDwei** [comment]
@xuyan333,@zhangweiqiang285. This is my opinion. In the m file, the size of images  is 2400*2000 . According to the paper, a column is a sample, there have 2000 samples, a sample contains 2400 data points and 2400*1 data is converted into 60*40  image.
The label is a label corresponding to each sample which indicates the type of fault. That is to say each type of image corresponds to the type of fault.
You can search the MNIST data set , You may understand more.

**Comment by qizhang00** [comment]
请问一下，看您的毕业论文，发现您是直接用的1D信号去做处理的，但是.m文件的matlab程序怎么感觉是形成二维图像呢？西储大学的轴承数据集已经下载了，但是对于您的数据集处理这一块纠结了很久，请指教！因为刚开始接触深度学习，言语中不专业之处，敬请谅解！

**Comment by everlast0430** [comment]
I really confused which one data you used exactly, please give me a hint..

**Comment by CQUPanda** [comment]
你是指哪组数据？






------------------ 原始邮件 ------------------
发件人: "itdojust1"<notifications@github.com>;
发送时间: 2018年11月11日(星期天) 中午11:31
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: ""<562251022@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning]About the original experiments data (#1)




I really confused which one data you used exactly, please give me a hint..

**Comment by everlast0430** [comment]
sorry, I wanna run image_matrix.m and I'am reading paper2 named 'Bearings Fault Diagnosis Based on Convolutional Neural Networks with 2-D Representation of Vibration Signals as Input' 
So, how to convert raw data from Case Western Reverse bearing into image using image_matrix..?
Can I get a sample data for running 'image_matrix.m' file?
thanks to reply!!.

**Comment by CQUPanda** [comment]
关于如何将一维振动信号转换成图像信号，这个实现比较简单，图像信号是二维矩阵的，振动信号是一维向量（假设为一维行向量），多个一维行向量进行拼接，就变成矩阵了。雷亚国的论文有讲。




------------------ 原始邮件 ------------------
发件人: "itdojust1"<notifications@github.com>;
发送时间: 2018年11月12日(星期一) 上午9:39
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: ""<562251022@qq.com>; "Comment"<comment@noreply.github.com>; 
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning]About the original experiments data (#1)




sorry, I wanna run image_matrix.m and I'am reading paper2 named 'Bearings Fault Diagnosis Based on Convolutional Neural Networks with 2-D Representation of Vibration Signals as Input'
 So, how to convert raw data from Case Western Reverse bearing into image using image_matrix..?
 Can I get a sample data for running 'image_matrix.m' file?
 thanks to reply!!.

**Comment by chenglin1996** [comment]
您好，我已经把西储大学的轴承数据集已经下载了，但是运行MATLAB程序一直运行不同，一直都是下面这个出错误：
错误使用 load
无法读取文件 'D:\360安全浏览器下载\tensorflow所有程序\Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master\image_matrix1': 没有此类文件或目录。
 
>> 
您能帮我看看是怎么回事吗？麻烦您了，期待您的回复

**Comment by QiuDwei** [comment]
@chenglin1996, firstly, it notes the image_matrix1 is abscent in Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master folder, then you may paste the location of the error code，such as the line ** .

**Comment by chenglin1996** [comment]
> @chenglin1996, firstly, it notes the image_matrix1 is abscent in Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master folder, then you may paste the location of the error code，such as the line ** .



> @chenglin1996, firstly, it notes the image_matrix1 is abscent in Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master folder, then you may paste the location of the error code，such as the line ** .

您好，谢谢您的回复，首先，我确实把数据集 image_matrix1放在了 Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master 文件夹下，我确实不知道地址方面哪里有问题了。

**Comment by chenglin1996** [comment]
您好，谢谢您的回复，首先，我确实把数据集 image_matrix1放在了 Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master 文件夹下，我确实不知道地址方面哪里有问题了。




------------------ 原始邮件 ------------------
发件人: "QiuDwei"<notifications@github.com>;
发送时间: 2019年5月11日(星期六) 中午1:15
收件人: "ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning"<Mechanical-Fault-Diagnosis-Based-on-Deep-Learning@noreply.github.com>;
抄送: "候鸟飞回来"<1834645316@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [ZhangWei1993/Mechanical-Fault-Diagnosis-Based-on-Deep-Learning]About the original experiments data (#1)




@chenglin1996, firstly, it notes the image_matrix1 is abscent in Mechanical-Fault-Diagnosis-Based-on-Deep-Learning-master folder, then you may paste the location of the error code，such as the line ** .

**Comment by QiuDwei** [comment]
@chenglin1996, You can send me the screenshot of the modified code and the location of the data set through mail, it is a folder problem. mail: qiudawei0320@163.com
